tests/Dockerfile

```diff
@@ -6,17 +6,13 @@ RUN dnf -y update && \
         american-fuzzy-lop \
         bind-utils \
         bodhi-client \
-        bzip2 \
```

Comment:
There is no harm if the specified packages already exist, and it's always better to state explicitly what we need than making assumptions on what the base image provides.  This is fine for required packages like util-linux, but not for optional packages like curl. Please drop this commit.